benchmarks/benchmarks/experimental/ugrid/regions_combine.py

```diff
@@ -183,7 +183,6 @@ def time_compute_data(self, n_cubesphere):
         _ = self.recombined_cube.data
 
     # Vulnerable to noise, so disabled by default.
```

Comment:
You left the comments behind!

Reply:
Good spot !
https://github.com/SciTools/iris/pull/5959/commits/1b2a9f140a0dfeeda142c1cdad217cece8790c99 fixes, I think